Code/Editor/Platform/Mac/gui_info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>CFBundleExecutable</key>
-	<string>Editor</string>
+	<string>EditorDummy</string>
```

Comment:
Isnt this one used for the Editor too?

Reply:
Yes, sort of. This info.plist will go into the Editor.app bundle. What this does is tell Finder(or Launchpad or Xcode) that when someone tries to launch this *bundle*, run the "EditorDummy" executable. "EditorDummy" will then launch the actual Editor. So, the CFBundleExecutable field in the info.plist only applies to what happens when you launch the bundle. We can still launch individual executables from terminal(or even from Finder if we use "show package contents" and navigate to the MacOS folder and launch the Editor executable from there).

Reply:
ah, gotcha, thanks for the details